vault/resource_ldap_auth_backend.go

```diff
@@ -118,7 +118,13 @@ func ldapAuthBackendResource() *schema.Resource {
 				return strings.Trim(v.(string), "/")
 			},
 		},
-
+		"local": {
```

Comment:
Please could we have a test that covers this attribute for LDAP auth as well? I think you should be able to add to `testLDAPAuthBackendCheck_attrs` and `testLDAPAuthBackendConfig_basic` to get coverage.

Reply:
Can do. I added a parameter to `Config_basic` and checked the state against the Vault API in `Check_attrs`.

```
=== RUN   TestLDAPAuthBackendGroup_import
--- PASS: TestLDAPAuthBackendGroup_import (0.22s)
=== RUN   TestLDAPAuthBackendGroup_basic
--- PASS: TestLDAPAuthBackendGroup_basic (0.19s)
=== RUN   TestLDAPAuthBackend_import
--- PASS: TestLDAPAuthBackend_import (0.20s)
=== RUN   TestLDAPAuthBackend_basic
--- PASS: TestLDAPAuthBackend_basic (0.70s)
=== RUN   TestLDAPAuthBackend_tls
--- PASS: TestLDAPAuthBackend_tls (0.71s)
=== RUN   TestLDAPAuthBackendUser_import
--- PASS: TestLDAPAuthBackendUser_import (0.20s)
=== RUN   TestLDAPAuthBackendUser_basic
--- PASS: TestLDAPAuthBackendUser_basic (0.20s)
=== RUN   TestLDAPAuthBackendUser_noGroups
--- PASS: TestLDAPAuthBackendUser_noGroups (0.19s)
=== RUN   TestLDAPAuthBackendUser_oneGroup
--- PASS: TestLDAPAuthBackendUser_oneGroup (0.19s)
PASS
ok      github.com/hashicorp/terraform-provider-vault/vault     3.122s
```